rpc/rpcutil/transaction.go

```diff
@@ -423,9 +451,15 @@ func MakeUnsignedSplitAddrTx(
 func MakeUnsignedTokenIssueTx(
 	wa service.WalletAgent, issuer, owner string, tag *rpcpb.TokenTag, fee uint64,
 ) (*types.Transaction, uint32, []*rpcpb.Utxo, error) {
+	if tag.GetDecimal() < 0 || tag.GetDecimal() > maxDecimal {
+		return nil, 0, nil, txlogic.ErrInvalidArguments
+	}
+	if uint64(tag.GetSupply()) > math.MaxUint64/uint64(math.Pow10(int(tag.GetDecimal()))) {
+		return nil, 0, nil, txlogic.ErrInvalidArguments
+	}
 	utxos, err := wa.Utxos(issuer, nil, fee)
 	if err != nil {
-		return nil, 0, nil, err
+		return nil, 0, nil, txlogic.ErrInvalidArguments
```

Comment:
why change to txlogic.ErrInvalidArguments here

Reply:
makeunsignedtx is the same error return